### ISSUE TITLE
Request is only allowed against master nodes.

### ISSUE THREAD

**Issue by pdepaepe** [Graylog2/graylog2-server#1470]
Hello,

Some API methods are only allowed against master node, such as simple GET dashboard.
This is quite annoying as it does not allow to horizontally scale the API behind a load balancer.

Is there a reason or a configuration i'm missing to not support it?

Thanks

**Comment by pdepaepe** [comment]
As a consequence of this bottleneck, a dashboard with 15 widgets starts to become a problem. 

Enabling react debugger on it, i see that all widget are called simultaneously, and if some of them are solved, following gives me

""Error loading widget value: API call failed GET http://@xxxxxxxx:12900/dashboards/56177f9760b2dc99c779cf93/widgets/f537f884-e83c-478e-82f3-12ab83fae815/value""

And then others

"Error loading widget value: \n\n\n\n\n<!DOCTYPE html>\n\n<html>\n    <head>\n        <title>Graylog - Disconnected</title>\n        <meta name=\"robots\" content=\"noindex, nofollow\">\n        <link rel=\"shortcut icon\" href=\"/graylog/assets/images/favicon.png\">\n        \n        <link rel=\"stylesheet\" media=\"screen\" href=\"/graylog/assets/stylesheets/bootstrap.min.css\">\n        <link rel=\"stylesheet\" media=\"screen\" href=\"/graylog/assets/stylesheets/newfonts.css\">\n        <link rel=\"stylesheet\" media=\"screen\" hr..."

Consequence on the graylog-master:

2015-10-14T15:13:54.113Z WARN  [NodePingThread] Did not find meta info of this node. Re-registering.

So for me, react is asking 15 widget in the same time, graylog master try to handle them all, but starts to reject some, then hang and re-register in the following.

User see on this page red pop up, and with a little luck can finally see charts later after some re-polling.

**Comment by pdepaepe** [comment]
I precise: all my graylog nodes share the same NTP.
```
WARN [NodePingThread] Did not find meta info of this node. Re-registering. 
```
Line appears only when i access the dashboard.

**Comment by kroepke** [comment]
This problem should be fixed in 2.0 alpha 1 and later.
AFAIK the restriction was because of state mutation and has been removed after refactoring the code.

Can you please reopen the issue if 2.0 alphas still contain the problem you are seeing?

Thanks!
